bleachbit/Cleaner.py

```diff
@@ -497,7 +501,8 @@ def __init__(self):
             # the uninstallers for software updates.
             self.add_option('updates', _('Update uninstallers'), _(
                 'Delete uninstallers for Microsoft updates including hotfixes, service packs, and Internet Explorer updates'))
-
+        elif 'posix' == os.name and sys.platform.startswith('darwin'):
```

Comment:
While this adds the option (the checkbox), I do not see a corresponding code to actually clean the recycle bin